### PR TITLE
[skin.estuary/pvr] Improve display of genre in PVR info dialog.

### PR DESCRIPTION
**Pull request by ksooo**
Followup to #16728 

Before #16728 (no genre displayed)

screenshot000

After #16728, before this PR (genre right to episode title, reducing space for episode title, which often leads to cut-off episode titles)

screenshot001

After this PR (genre displayed along with other extended data)

screenshot001

### CODE REVIEW

**Review by phunkyfish — APPROVED** [review]
Looks good!

File: addons/skin.estuary/xml/Includes_PVR.xml [hunk 1]
```diff
@@ -209,11 +209,13 @@
 					<font>font45</font>
 					<label>$INFO[ListItem.ChannelName]</label>
 				</control>
-				<control type="label">
+				<control type="textbox">
 					<top>60</top>
-					<height>200</height>
+					<height>130</height>
```
Comment (phunkyfish) [inline addons/skin.estuary/xml/Includes_PVR.xml]: Just curious, why the change in height?

Comment (ksooo) [reply addons/skin.estuary/xml/Includes_PVR.xml]: Fixes a bug that was there before, but never got triggered before your PR. The height covers 4 lines, progress is drawn over last line, if 4 items are to be displayed 
and a progress bar => 

screenshot002

... and moving the bar down a bit is not really an option, because there is another bar, shown for in progress shows, that has text labels below the bar graph. Making heights dynamic according the different scenarios clutters the skin xml code in a way that I decided to keep it simple.

Comment (DaVukovic) [reply addons/skin.estuary/xml/Includes_PVR.xml]: Probably we now have one more infolabel and if all are shown it might not match with the progressbar below. Might have been a bug before already.

**Comment by ronie** [comment]
looks good, though i'd prefer a multiline label over a scrolling textbox.

**Comment by ksooo** [comment]
Problem the varying number of items to display. and limited space on screen. Thus, I've chosen a textbox which scrolls when there are more items to display than lines available.

If you have an idea for a better implementation, you are, like always, very welcome. My skiing should are very limited, as you know.

**Comment by ksooo** [comment]
> i think i would drop the Date line from the list. it's already shown on the left side.

 There can be more items - 5 max. And in the future there might be even more of them. So, dropping "Date" will not really solve the problem.

 So, I guess I will stick with the textbox, if this is okay for you?!

**Comment by ronie** [comment]
if it was up to me, i wouldn't try to show that many infolabels in this view.

also, keep in mind the plot at the bottom can potentially scroll as well, which would result in two scrolling elements on the screen... meh.

**Comment by ksooo** [comment]
@ronie You are right. I have already seen those two scrolling areas in action and it looks, well... not good. We need another solution for the "genre" problem. And I'm also with you not to put to many info into the PVR panel which is supposed to give a quick an overview only. For details, we have the PVR info dialog.

**Comment by phunkyfish** [comment]
For many users (such as those who use a CEC remote) they never enter the PVR info dialog from recordings and you cannot access it from timers. Essentially the dialog is only used in the Guide.

**Comment by phunkyfish** [comment]
Am I right in saying we could have expiration or genre but not both sensibly?

**Comment by ksooo** [comment]
@phunkyfish  @ronie what do you guys think about this approach?

screenshot002

Idea is to only have max. 3 items above the title - date, duration, genre. And to add "everything else" in the (scrolling) text box below the title. Thus, we only have one scrolling area, but can still display all available data.

**Comment by phunkyfish** [comment]
Cool, I like it.

**Comment by ksooo** [comment]
Here is another attempt to make the pvr info panel more usable / good looking:

I reordered the controls, most notably the titel/subtitle stiff is now at the top of the panel. I felt that the most imprtant information has to be on top, like we do at all (?) other dialogs/windows in Estuary...
screenshot009
screenshot010
screenshot011

**Comment by phunkyfish** [comment]
And would expiration time live down with plot?

**Comment by ksooo** [comment]
> And would expiration time live down with plot?

 yes.

**Comment by phunkyfish** [comment]
I like how it fits in with the style of the other screens.

**Comment by ronie** [comment]
@ksooo i don't think it's a good idea to merge changes without proper review by one of the team skinners.

**Comment by ksooo** [comment]
Ooops, sorry. I thought I got your okay already. My fault. Any review comments?

**Comment by ksooo** [comment]
(I only shuffled existing xml around, btw, no real changes here...)
